blueocean-web/gulpfile.js

```diff
@@ -43,7 +46,7 @@ builder.bundle('src/main/js/blueocean.js')
 //
 builder.bundle('src/main/js/try.js')
     .inDir('target/classes/io/jenkins/blueocean')
-    .withExternalModuleMapping('jquery-detached', 'core-assets/jquery-detached:jquery2') // Bundled in Jenkins 2.x 
+    .import('jquery-detached', 'core-assets/jquery-detached:jquery2') // Bundled in Jenkins 2.x
```

Comment:
`try.js` is used in classic Jenkins and doesn't use `react`. It's the blue button on the top bar.